src/main/java/frc/team2412/robot/commands/drivebase/DriveCommand.java

```diff
@@ -75,30 +79,34 @@ public void execute() {
 								- (speedLimiter.getAsDouble()
 										* (1 - triggerModifierEntry.getDouble(TRIGGER_MODIFIER_DEFAULT))));
 
-		double x = deadbandCorrection(-forward.getAsDouble());
-		double y = deadbandCorrection(strafe.getAsDouble());
-		double rot = deadbandCorrection(-rotation.getAsDouble());
+		if (autoBalance.getAsBoolean()) {
+			drivebaseSubsystem.drive(0, 0, new Rotation2d(), fieldOrientedEntry.getBoolean(true), true);
+		} else {
+			double x = deadbandCorrection(-forward.getAsDouble());
+			double y = deadbandCorrection(strafe.getAsDouble());
+			double rot = deadbandCorrection(-rotation.getAsDouble());
 
-		// math for normalizing and cubing inputs
-		double magnitude = Math.pow(Math.min(Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2)), 1), 3);
-		double angle = Math.atan2(y, x);
-		double cubed_x = magnitude * Math.cos(angle);
-		double cubed_y = magnitude * Math.sin(angle);
+			// math for normalizing and cubing inputs
+			double magnitude = Math.pow(Math.min(Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2)), 1), 3);
+			double angle = Math.atan2(y, x);
+			double cubed_x = magnitude * Math.cos(angle);
+			double cubed_y = magnitude * Math.sin(angle);
 
-		drivebaseSubsystem.drive(
-				(cubeSpeedEntry.getBoolean(false) ? cubed_x : x)
-						* driveSpeedModifier
-						* DrivebaseSubsystem.MAX_DRIVE_SPEED_METERS_PER_SEC, // convert from percent to m/s
-				(cubeSpeedEntry.getBoolean(false) ? cubed_y : y)
-						* driveSpeedModifier
-						* DrivebaseSubsystem.MAX_DRIVE_SPEED_METERS_PER_SEC,
-				Rotation2d.fromRotations(
-						rot
-								* rotationSpeedEntry.getDouble(1.0)
-								* DrivebaseSubsystem.MAX_ROTATIONS_PER_SEC
-										.getRotations()), // convert from percent to rotations per second
-				fieldOrientedEntry.getBoolean(true),
-				false);
+			drivebaseSubsystem.drive(
+					(cubeSpeedEntry.getBoolean(false) ? cubed_x : x)
+							* driveSpeedModifier
+							* DrivebaseSubsystem.MAX_DRIVE_SPEED_METERS_PER_SEC, // convert from percent to m/s
+					(cubeSpeedEntry.getBoolean(false) ? cubed_y : y)
+							* driveSpeedModifier
+							* DrivebaseSubsystem.MAX_DRIVE_SPEED_METERS_PER_SEC,
+					Rotation2d.fromRotations(
+							rot
+									* rotationSpeedEntry.getDouble(1.0)
```

Comment:
I am realizing that 1.0 seems pretty high for a default.  Seems better err on the lower side so it doesn't accidentally zoom out of control and hurt someone or itself.